src/checkers/inference/InferenceQualifierHierarchy.java

```diff
@@ -325,7 +325,7 @@ public AnnotationMirror getTopAnnotation(final AnnotationMirror am) {
             return varAnnot;
         } // else
 
-        return this.getTopAnnotations().iterator().next();
+        return inferenceMain.getRealTypeFactory().getQualifierHierarchy().getTopAnnotations().iterator().next();
```

Comment:
There's a TODO comment in line 319 above that needs an update.

Reply:
This changes makes getTopAnnotation and getBottomAnnotation consistent, which seems like a good thing.
I'm not sure what update to that TODO comment would improve the situation.